src/pages/home/report/ReportActionItemParentAction.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {View} from 'react-native';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import ONYXKEYS from '../../../ONYXKEYS';
+import styles from '../../../styles/styles';
+import OfflineWithFeedback from '../../../components/OfflineWithFeedback';
+import * as Report from '../../../libs/actions/Report';
+import reportPropTypes from '../../reportPropTypes';
+import * as StyleUtils from '../../../styles/StyleUtils';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../../components/withWindowDimensions';
+import compose from '../../../libs/compose';
+import withLocalize from '../../../components/withLocalize';
+import ReportActionItem from './ReportActionItem';
+import reportActionPropTypes from './reportActionPropTypes';
+
+const propTypes = {
+    /** The id of the report */
+    reportID: PropTypes.string.isRequired,
+
+    /** The id of the parent report */
+    // eslint-disable-next-line react/no-unused-prop-types
+    parentReportID: PropTypes.string.isRequired,
+
+    /** ONYX PROPS */
+
+    /** The report currently being looked at */
+    report: reportPropTypes,
+
+    /** The actions from the parent report */
+    // TO DO: Replace with HOC https://github.com/Expensify/App/issues/18769.
+    parentReportActions: PropTypes.objectOf(PropTypes.shape(reportActionPropTypes)),
+
+    ...windowDimensionsPropTypes,
+};
+const defaultProps = {
+    report: {},
+    parentReportActions: {},
+};
+
+const ReportActionItemParentAction = (props) => {
+    const parentReportAction = props.parentReportActions[`${props.report.parentReportActionID}`];
+    return (
+        <OfflineWithFeedback
+            pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(props.report, 'pendingFields.createChat')}
```

Comment:
do we do this lodash get anywhere else? If so maybe we should have a utils method?